Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:15-jdk-alpine
+VOLUME /tmp
+ADD build/libs/gateway-0.0.1-SNAPSHOT.jar app.jar
```

Comment:
It wil stop working if version changed

Reply:
1. Edited the dockerfile.
When changing the jar file version, the image is always created.
2. Added instructions for creating and running an image in Readme.md.